targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF53/objects.h

```diff
@@ -120,7 +120,6 @@ struct i2c_s {
 };
 
 struct analogin_s {
-    uint8_t channel;
 };
 
```

Comment:
Is this still needed?

Reply:
The struct? Yes as required to add per target